src/js/components/HeaderBar.tsx

```diff
@@ -1,20 +1,67 @@
 import * as React from "react";
+import AuthStore from "#SRC/js/stores/AuthStore";
+
 import { HeaderBar as UIHeaderBar } from "ui-kit-stage/HeaderBar";
+import AccountDropdown from "./AccountDropdown";
+
+function getUserAccountDropdown() {
```

Comment:
🎨 I'd love to see this moved to its own component, `HeaderBar` should really only be the bar itself 🤔  (applies to all of the added code here)

Reply:
`HeaderBar.tsx` is using `UIHeaderBar`, which is just the bar itself.

Reply:
I know, but I think we will drop the `UiHeaderBar` component…

Reply:
What do you mean drop? I thought we are just moving it into the right folder in dcos-ui. https://jira.mesosphere.com/browse/DCOS-39074

Reply:
Fair enough. We can do all this for DCOS-39074.